Source/DiabloUI/selgame.cpp

```diff
@@ -421,11 +421,11 @@ static bool IsGameCompatible(GameData *data)
 	return false;
 }
 
-void selgame_Password_Select(int value)
+void selgame_Password_Select(int /*value*/)

```

Comment:
Did it really complain about an unused function parameter having a name? That's not a very helpful rule, IMHO.

Is is for some reason not possible to remove this parameter if it's unused?